webui/react/src/utils/data.ts

```diff
@@ -149,3 +149,31 @@ export const isEqual = (a: unknown, b: unknown): boolean => {
   if (a === b) return true;
   return JSON.stringify(a) === JSON.stringify(b);
 };
+
+// export const getSubObjectCopy = (obj: RawJson, accessors: string[]): any => {
+//   return accessors
+//     .reduce((acc, cur) => acc === undefined ? acc : acc[cur], obj);
```

Comment:
**nit:** if `obj` is passed in as the starting point for the reducer, the final output will be the actual sub object instead of a copy

Reply:
yeah you're right yet I was having some issue with this. I can give it another try it look nicer.